engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -2294,7 +2292,7 @@ private void orchestrateMigrateWithStorage(final String vmUuid, final long srcHo
                 s_logger.info("Migration was unsuccessful.  Cleaning up: " + vm);
                 _alertMgr.sendAlert(alertType, srcHost.getDataCenterId(), srcHost.getPodId(),
                         "Unable to migrate vm " + vm.getInstanceName() + " from host " + srcHost.getName() + " in zone " + dc.getName() + " and pod " + dc.getName(),
-                        "Migrate Command failed.  Please check logs.");
+                        "Migrate Command failed. HA may not work for Hyper-V VM.  Please check logs.");
```

Comment:
A hypervisor specific message in a generic module can lead to confusion. I would like to see the Hyper-V text replaced by a reference to hypervisortype or replaced by something more generic altogether